crypto/ec/ec_lib.c

```diff
@@ -746,9 +746,13 @@ void EC_POINT_free(EC_POINT *point)
     if (point == NULL)
         return;
 
+#ifndef FIPS_MODULE
     if (point->meth->point_finish != 0)
```

Comment:
Is this check really not needed in the FIPS provider?

Reply:
Ah no, you are right. What confused me was `0` rather than `NULL`, so I didn't notice immediately that this was calling a function pointer